src/App.js

```diff
@@ -1,10 +1,12 @@
 import React, { useState } from 'react';
 import './App.css';
-
 import Board from './components/Board';
+import matthew from './img/matthewface.png';
+import junior from './img/juniorface.jpg';
 
-const PLAYER_1 = 'X';
-const PLAYER_2 = 'O';
+const PLAYER_1 = <img src={matthew} alt="X" />;
+const PLAYER_2 = <img src={junior} alt="O" />;
```

Comment:
In the future, consider having further separation between data and presentation. Keeping the variables as `X` and `O` and changing only the rendering logic to show the kitties may be a bit cleaner. It also would have allowed you to still pass the automated tests.